codeql.groovy

```diff
@@ -32,10 +32,6 @@ data.each { release ->
     }
     def bundleDate = (tagname =~ /codeql-bundle-(.*)/)[0][1]
 
-    if(bundleDate.toInteger() < 20200826) {
```

Comment:
is this removing a necessary check though? I'm guessing it's trying to filter out some old versions.

why is it failing now

Reply:
I didn't saw anything weird in the resulting JSON when removing this cast.

Reply:
I could have filtered out the `v2.6.0-beta.1` version (which has a different URL pattern) and keep the test but I don't think it's useful.

Reply:
Not sure why I don't retrieve locally [the older versions still available on GitHub](https://github.com/github/codeql-action/releases?page=5), readding the filter.

Reply:
@timja [The GitHub API](https://api.github.com/repos/github/codeql-action/releases) return only the releases created on the 2021-05-03 or after ([last one](https://github.com/github/codeql-action/releases/tag/codeql-bundle-20210503)), while [the GitHub releases page](https://github.com/github/codeql-action/releases?page=5) returns older releases. 🤷 

Should I add back the filtering anyway? (taking care of the exception)

Reply:
No that’s fine then